examples/typescript/utils.ts

```diff
@@ -21,11 +24,31 @@ export function compilePackage(
     console.log("aptos is not installed. Please install it from the instructions on aptos.dev");
   }
 
-  const addressArg = namedAddresses.map(({ name, address }) => `${name}=${address}`).join(" ");
+  const addressArg = namedAddresses.map(({ name, address }) => `${name}=${address.toString()}`).join(" ");
```

Comment:
I think this may need to be joined with commas.  Also, the toString() should work automatically as a `toString()`
```suggestion
  const addressArg = namedAddresses.map(({ name, address }) => `${name}=${address.toString()}`).join(",");
```